gulpfile.js

```diff
@@ -7,9 +7,10 @@ const gulp = require('gulp'),
     addsrc = require('gulp-add-src'),
     modularBuild = require('./modular-build'),
     mochaPhantomJS = require('./mocha-phantomjs'),
-    fs = require('fs');
-
-
+    fs = require('fs'),
+    through = require('through2'),
+    zlib = require('zlib');
```

Comment:
We don't need to update the packages.json for these modules, do we?

Reply:
No, we don't need. **through2** is the dependency which we used in modular-build.js and **zlib** is the built-in package of Node.js.

Reply:
OK, that's great.